apps/src/templates/instructions/teacherFeedback/TeacherFeedbackTab.jsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import TeacherFeedback from '@cdo/apps/templates/instructions/teacherFeedback/TeacherFeedback';
+import ReadonlyTeacherFeedback from '@cdo/apps/templates/instructions/teacherFeedback/ReadonlyTeacherFeedback';
+import {
+  teacherFeedbackShape,
+  rubricShape
+} from '@cdo/apps/templates/instructions/teacherFeedback/types';
+import PropTypes from 'prop-types';
+
+const TeacherFeedbackTab = ({
+  teacherViewingStudentWork,
+  displayReviewTab,
+  visible,
+  rubric,
+  latestFeedback,
+  token,
+  serverScriptId,
+  serverLevelId,
+  teacher,
+  innerRef
+}) => {
+  const renderEditableFeedback = teacherViewingStudentWork && !displayReviewTab;
+  const renderReadonlyFeedback =
+    !teacherViewingStudentWork && (!!rubric || !!latestFeedback);
```

Comment:
I think a comment describing the intended behavior in prose here might be helpful.